### PR TITLE
Added comments to one of public class - CodableConceptInfo

### PR DESCRIPTION
**Pull request by gauravgupta98**
## Added comments to CodableConceptInfo class.
I think we should add comments to our public methods/classes etc.
For now, I've just added comments to single class. If we are good to go with this proposed change then I can go ahead and pick other public facing methods and improve the coding standards.

## FHIR Team Checklist
- [x] **Update the title** of the PR to be succinct and less than 50 characters
- [ ] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- [ ] Tag the PR with the type of update: **Bug**, **Dependencies**, **Enhancement**, or **New-Feature**
- [ ] Tag the PR with **Azure API for FHIR** if this will release to the managed service
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

**Comment by brendankowitz** [comment]
Hi @gauravgupta98, yes I think this makes sense. The more urgent task relating to docs is fixing the existing errors that are blocking generation: #1681

**Comment by gauravgupta98** [comment]
Hi @brendankowitz, thanks for the review. I'll have a look at the issue you mentioned. Also, If we are good on this change, can we go ahead and complete the PR?

File: src/Microsoft.Health.Fhir.Core/Models/CodableConceptInfo.cs [hunk 1]
```diff
@@ -12,18 +12,28 @@ namespace Microsoft.Health.Fhir.Core.Models
...
+        /// <param name="coding">The Coding collection.</param>
         public CodableConceptInfo(IEnumerable<Coding> coding)
         {
             EnsureArg.IsNotNull(coding);
 
             Coding = coding.ToList();
         }
 
+        /// <summary>
+        /// Gets the collection of Coding.
```
Comment (brendankowitz) [inline src/Microsoft.Health.Fhir.Core/Models/CodableConceptInfo.cs:35]: nit: it think this reads better as `Gets the Coding collection` or `Gets the collection of Codings`

Comment (gauravgupta98) [reply src/Microsoft.Health.Fhir.Core/Models/CodableConceptInfo.cs:35]: Shouldn't the property name be Codings? What are your thoughts?

**Review by brendankowitz — APPROVED** [review]

**Comment by gauravgupta98** [comment]
Looks like some tests are failing. @brendankowitz can you please have a look. Thanks

**Pull request merged**